Makefile

```diff
@@ -113,34 +107,41 @@ checkmigname:
 # Migration operations
 ## Rollback the last migration
 .PHONY: rollback
-rollback: $(GOOSE) checkdbvars
+rollback: checkdbvars
 	cd db/migrations;\
-	  $(GOOSE) postgres "$(CONNECT_STRING)" down
-	pg_dump -O -s $(CONNECT_STRING) > db/schema.sql
-
+	  $(GOOSE) -table "maker.goose_db_version" postgres "$(CONNECT_STRING)" down
+	pg_dump -n 'maker' -n 'api' -O -s $(CONNECT_STRING) > db/schema.sql
 
 ## Rollback to a select migration (id/timestamp)
 .PHONY: rollback_to
-rollback_to: $(GOOSE) checkmigration checkdbvars
+rollback_to: checkmigration checkdbvars
 	cd db/migrations;\
-	  $(GOOSE) postgres "$(CONNECT_STRING)" down-to "$(MIGRATION)"
+	  $(GOOSE) -table "maker.goose_db_version" postgres "$(CONNECT_STRING)" down-to "$(MIGRATION)"
 
 ## Apply all migrations not already run
 .PHONY: migrate
-migrate: $(GOOSE) checkdbvars
+migrate: checkdbvars
+	psql $(NAME) -c 'CREATE SCHEMA IF NOT EXISTS maker;'
 	cd db/migrations;\
-	  $(GOOSE) postgres "$(CONNECT_STRING)" up
-	pg_dump -O -s $(CONNECT_STRING) > db/schema.sql
+	  $(GOOSE) -table "maker.goose_db_version" postgres "$(CONNECT_STRING)" up
+	pg_dump -n 'maker' -n 'api' -O -s $(CONNECT_STRING) > db/schema.sql
```

Comment:
Not a responsibility of this PR but I wonder if we should create a story to remove/rename the `api` schema - seems like that could get confusing with multiple plugins + every schema include in the actual API 🤔

Reply:
I can add a story to Jira if you want. API definitely seems confused.